facturacion_mexico/tests/test_pdf_custom_section.py

```
@@ -0,0 +1,198 @@
+from unittest.mock import MagicMock, patch
+
+import frappe
+from frappe.tests import IntegrationTestCase
+
+from facturacion_mexico.facturacion_fiscal.timbrado_api import (
+	_build_pdf_custom_section,
+	render_pdf_note_template,
+)
+
+
+def _mock_settings(**kwargs):
```

Ruff (0.15.17)

[warning] Missing return type annotation for private function `_mock_settings`

(ANN202)